doc/API/Devices.md

```diff
@@ -564,6 +564,36 @@ Output:
 
 Output is an image.
 
+### `get_graph_by_service`
+
+Get the graph for a service
+
+Route: `/api/v0/devices/:hostname/services/:service_id/graphs/:datasource`
+
+- hostname can be either the device hostname or id
+- service id
+- datasource is the name of the service datasource
+
```

Comment:
Missing the title `Input:` here to allow people to specify the below values.

Reply:
I've updated the docs.